fleetscorebot/package.json

```diff
@@ -6,18 +6,28 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "prod": "NODE_ENV=production node index.js get",
-    "staging": "NODE_ENV=staging node index.js get"
+    "staging": "NODE_ENV=staging node index.js get",
+    "precommit": "pretty-quick --staged"
   },
   "author": "Gergely Imreh <imrehg@gmail.com>",
```

Comment:
I think this is supposed to be resin.io now, but again there's no rush.

Reply:
@CameronDiver you mean the author? Because on github my main username is this one, not sure how's the attribution works, would need to look into that....

Reply:
There's recent changes about this, but don't worry about it for this PR. (and this is -playground)